torch_xla/csrc/convolution.cpp

```diff
@@ -121,16 +124,15 @@ xla::XlaOp PadInputFromOutputSize(xla::XlaOp input,
   return PadToSize(input, expected_input_sizes);
 }
 
-// Create a TF convolution metadata structure out of PyTorch convolution
-// attributes.
-tensorflow::ConvOpAttrs MakeConvOpAttrs(
-    absl::Span<const int64_t> spatial_stride,
-    absl::Span<const int64_t> spatial_padding,
-    absl::Span<const int64_t> spatial_dilation, bool depthwise) {
+// Create PTXLAConvOpAttrs
+PTXLAConvOpAttrs MakeConvOpAttrs(absl::Span<const int64_t> spatial_stride,
```

Comment:
ditto, just leave it as `ConvOpAttrs`, We almost never prefix any helper function with `PTXLA`

Reply:
Thanks, Jack, updated